### PR TITLE
increase DB pool size to avoid too many new connections.

### PR DESCRIPTION
**Pull request by yuyiguo**

### CODE REVIEW

File: apps/base/rucio-daemons/cms-rucio-daemons.yaml [hunk 1]
```diff
@@ -76,6 +76,8 @@ conveyorPoller:
   podAnnotations:
     prometheus.io/scrape: "true"
     prometheus.io/port: "8080"
+  database:
+    pool_size: 10
```
Comment (ericvaandering) [inline apps/base/rucio-daemons/cms-rucio-daemons.yaml:80]: You need to put this under a config heading (see the Kronos example I sent you yesterday)

File: apps/base/rucio-server/cms-rucio-server.yaml [hunk 2]
```diff
@@ -106,9 +106,11 @@ config:
       username: "cmsrucio"
       #config.trace.tracedir: directory where traces are written to (default "/var/log/rucio/trace")
       tracedir: "/var/log/rucio/trace"
+  server:
```
Comment (ericvaandering) [inline apps/base/rucio-server/cms-rucio-server.yaml:109]: You don't need server here, I think. just config.database.pool_size

**Review by ericvaandering — CHANGES_REQUESTED** [review]

File: apps/base/rucio-server/cms-rucio-server.yaml (hunk 2 above)
Comment (yuyiguo) [reply apps/base/rucio-server/cms-rucio-server.yaml:109]: where to config.database.pool_size? 
Another question: there are sessions "tracer_kronos" vs "tracerKronos". How are the configs divided into these two sessions? how the section names are decided?

Comment (ericvaandering) [reply apps/base/rucio-server/cms-rucio-server.yaml:109]:
```
config:   
   database:
      pool_size:
```      

Anything in `config:` goes into rucio.cfg and uses the section names there. `tracerKronos` is for the k8s Deployment and includes pod parameters, execution parameters, etc.

**Pull request merged**